tests/test_deps.py

```
@@ -36,18 +36,12 @@
 	raise ImportError("[CWE-758] Failed to import test context") from _cause
 
 try:
-	if 're' not in sys.modules:
-		import re
-	else:  # pragma: no branch
-		re = sys.modules["""re"""]
+	import re
 except Exception as _cause:  # pragma: no branch
 	raise ImportError("[CWE-440] re Failed to import.") from _cause
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)